Tutorials/drawing-options/drawing-options-functions-using-bitmap.md

```diff
@@ -0,0 +1,176 @@
+---
+title: "Drawing Options functions using DrawBitmap"
+description: Learn how to use drawing options functions in Splashkit and combine them to create dynamic and visually stunning graphics.
+author: Sharvani Kandala
+date: 05/05/2024
+tags: programming, graphics, Splashkit, tutorials.
+---
+
+## Window Setup
+
+To begin utilizing drawing options functions in Splashkit, set up the game window as follows:
+
+1. Import the necessary Splashkit headers at the beginning of your code:
+
+   ```cpp
+   #include "splashkit.h"
+   ```
+
+2. In the `main` function, configure the game window:
+
+   ```cpp
+    int main()
+    {
+       open_window("Drawing", 800, 600);
+       return 0;
+    }
+    ```
+
+3. Function to Draw the Bitmap:
+
+   ```cpp
+   void draw_bitmap(bitmap bmp, double x, double y, drawing_options opts);
+   ```
+
+- `bmp`: The bitmap to draw.
+- `x`: The x-coordinate for the top-left corner of the bitmap.
+- `y`: The y-coordinate for the top-left corner of the bitmap.
+- `opts`: Drawing options to apply.
+
+## Drawing Options Functions
+
+## Example 1: Applying Option Part Bmp
```

Comment:
This heading should be H3 (### instead of ##) as it is in the group of the heading above.